Servlets/web/index.html

```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: youli
+  Date: 13.05.2019
+  Time: 12:51
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+  <title>Moment</title>
+</head>
+<body>
+  Java Web Application
+</body>
+</html>
```

Comment:
And one more general point regarding 8th lab
you should declare web.xml anyway  even with the stub content:
```
<?xml version="1.0" encoding="UTF-8"?>
<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
         version="4.0">
</web-app>
```
Practically there is many reasons technical and more "casual".
One simple one of them on others computers Idea will recognize that it is web project, but will not find web directory, so facet will not be configured correctly.